standards/async.md

```diff
@@ -0,0 +1,90 @@
+- Asynchronous requests **MUST** return 202 (Accepted) response status code when a long-running processing requests was successfully initiated.
+    - No other 2xx codes **SHOULD** be returned in response to the asynchronous request initialization, even if request has completed before the initiating request returns.
+- `Operation-Id` header is **MAY** be supported and can be provided as part of the request.
+- Process ID **SHOULD** be automatically generated and returned to the client if `Operation-Id` header is not provided.
+- Asynchronous request **MUST** have a response body that extends from the following schema:
+```json
+    {
+        "id": "string",                                                                 // required: unique id representing the resource
+        "ref": "sps-ref",                                                               // optional
+        "status": "SUCCEEDED" | "FAILED" | "RUNNING" | "NOT_STARTED" | "CANCELED",      // required
+        "detail": "same as detail from bulk... maybe error message? or complex object?",// required
+        "result": "",                                                                   // optional
+        "completenessDescription": "percentage?",                                       // optional
+        "createdDateTime": "",                                                          // required
+        "createdBy": "" | null,                                                         // optional
+        "completedDateTime": "" | null,                                                 // required
+        ...EXTENDED...
+    }
+```
+
+Example workflow for an asynchronous resource creation request:
+
+```
+// REQUEST
+POST /reports                               // initiates long running processing by posting job with new parameters
+Operation-Id: f7cf8412-08ed-40c9-ac1b-296da9d1d970
+{
+    ...Custom Request Body Parameters
+}
+
+// RESPONSE
+202 ACCEPTED
+Operation-Location: /reports/f7cf8412-08ed-40c9-ac1b-296da9d1d970
+{
+    "id": "f7cf8412-08ed-40c9-ac1b-296da9d1d970",
+    "result": null
+}
+```
+
+Once new long-running request posted, corresponding instance of running process created for tracking purpose. To access details about process and completeness status GET operation **MUST** be used. Such request will return current status of the background process along with the request parameters.
+
+- Status of long-running processing **MUST** be always returned via GET request with provided `id` of the asynchronous process
+
+```
+// request to get details about already posted asynchronous request
+GET /reports/f7cf8412-08ed-40c9-ac1b-296da9d1d970
+
+// RESPONSE
+200 OK
+{
+    "id": "f7cf8412-08ed-40c9-ac1b-296da9d1d970",                                                               
+    "ref": "sps:report::f7cf8412-08ed-40c9-ac1b-296da9d1d970",
+    "status": "RUNNING",
+    "detail": "same as detail from bulk... maybe error message? or complex object?",
```

Comment:
same questions on if details on errors should be some standard error format? definitely this answer should match between this and bulk updates.